cfme/tests/containers/test_basic_metrics.py

```diff
@@ -8,6 +8,17 @@
     pytest.mark.tier(1),
     pytest.mark.provider([ContainersProvider], scope='function')]
 
+
+@pytest.fixture(scope="module")
+def reduce_metrics_collection_threshold(appliance):
+    appliance.ssh_client.put_file("scripts/openshift/change_metrics_collection_threshold.rb", "/var/www/miq/vmdb")
+    appliance.ssh_client.run_rails_command("change_metrics_collection_threshold.rb 5.minutes")
```

Comment:
Oh, no, I'm just suggesting that whatever we set it to, it shouldn't be hard coded in the code.

Reply:
It should be either a constant or config item.

Reply:
But I posed it as a question, because I don't feel super strongly about this, but I keep finding hard coded things in our code when we have config files and ways of not hard coding things.   Er, I have even recently created some and my conscious is burning (-;

Reply:
Yes, cfme_data.yaml would do.   But certainly defining a constant in the module and using that would be better that just having the data inline in the code.